calico/scripts/install-calico-windows.ps1

```diff
@@ -166,11 +166,31 @@ function GetBackendType()
             throw "{{site.prodname}} on Linux has IPIP enabled. IPIP is not supported on Windows nodes."
         }
 
+        # Check FelixConfig first.
         $encap=c:\k\kubectl.exe --kubeconfig="$KubeConfigPath" get felixconfigurations.crd.projectcalico.org default -o jsonpath='{.spec.vxlanEnabled}' -n $CalicoNamespace
         if ($encap -EQ "true") {
             return ("vxlan")
+        } elseif ($encap -EQ "false") {
+            return ("bgp")
+        } else {
+           # If any IPPool has IPIP enabled, we need to exit the installer. The
```

Comment:
Isn't this possibly a breaking change? Existing clusters might have IPIP pools on them and work fine because they don't select the Windows node?

Reply:
I thought that before the [deprecation](https://github.com/projectcalico/calico/pull/5576) of the encap fields in FelixConfiguration, if an IP pool with IPIP was created than the default FelixConfig would have `ipipEnabled: true`.

If that's not the case, then yeah this would be a breaking behaviour. I think this would be okay to release note and suggest that users explicitly tell the install script what networking backend to use. WDYT?

Reply:
I don't think it is a breaking change because:
1. If existing cluster has encap fields in FelixConfiguration, the code check it first before exam ippools.
2. If the cluster been upgraded to use ippool, the code will check all ippools to get either VXLAN or BGP.

Reply:
Yep, you're right. :shipit: